invoker/core/src/main/java/com/google/cloud/functions/invoker/BackgroundFunctionExecutor.java

```diff
@@ -333,17 +339,15 @@ public void service(HttpServletRequest req, HttpServletResponse res) throws IOEx
       res.setStatus(HttpServletResponse.SC_OK);
     } catch (Throwable t) {
       res.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
-      logger.log(Level.WARNING, "Failed to execute " + functionExecutor.functionName(), t);
+      logger.log(Level.SEVERE, "Failed to execute " + functionExecutor.functionName(), t);
```

Comment:
A lot of changes due to auto-formatting, so highlighting the logic change here.

Reply:
I agree with @grant's comments below. Could we split this into two PRs: one for simple formatting changes, one for the logging changes. It makes it easier to review, but will also make our lives simpler if a commit needs to get reverted later.